builtin/logical/aws/path_user.go

```diff
@@ -18,6 +18,12 @@ import (
 func pathUser(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "(creds|sts)/" + framework.GenericNameWithAtRegex("name"),
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixAWS,
+			OperationSuffix: "credentials|sts-credentials",
```

Comment:
creds vs credentials again, would say stick with what the path has

Reply:
The idea behind these is to give more human-readable names for the generated methods. The methods names will often deviate from the API path. I prefer not to abbreviate whenever possible to be inclusive of people who don't speak English natively.